datasette/my_example.py

```diff
@@ -22,9 +20,19 @@ def initialize_db(timestamp: datetime) -> datetime:
     return timestamp
 
 
+def run_aggregate_query() -> None:
+    log.info("Read SQL file")
+    with open("agg_query.sql", "r") as fd:
+        sqlFile = fd.read()
+    db = sqlite_utils.Database("laliga1819.db")
+    log.info("Executing SQL file on laliga db")
+    db.execute(sqlFile)
```

Comment:
to test this function we can create a "test" version of the database and pass to the function via a patch.
In addition we can test exception handling